API.md

```diff
@@ -20,6 +20,7 @@ The central ledger has two APIs targeted at different consumers. The DFSP API is
 * `POST` [**Get charge quote**](#get-a-charge-quote) 
 * `POST` [**Get authentication token**](#get-authentication-token)
 * `GET`  [**Health**](#health)
+* `GET`  [**Kafka**](#kafka)
 
 #### [Admin API](#admin-api) endpoints
```

Comment:
This is not for the ADMIN API only: remember that the Admin API run its own separated thread from the normal API. We need potentially both (but at this moment only care about the normal API) to be able to have some kind of management capability going forward for the Kafka consumers. The APIs are for different things. Admin API is for the administrating of the Ledger, where the normal API is for the client usage of the Ledger. 

The Kafka API that I am asking for is more of a backend service only for maintenance purposes only, and potentially impacts both.